app/models/user.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  attr_accessor :remember_token
+  before_create :remember_me
+
+  validates :user_name, :email, presence: true, uniqueness: true
```

Comment:
Would be a good idea to validate email format with an email regex, to avoid emails like joe@doe and others [OPTIONAL]